contactql/parser.go

```diff
@@ -14,17 +14,20 @@ import (
 	"github.com/shopspring/decimal"
 )
 
-type boolOp string
+// BoolOperator is a boolean operator (and or or)
+type BoolOperator string
 
 const (
-	boolOpAnd boolOp = "and"
-	boolOpOr  boolOp = "or"
+	// BoolOperatorAnd is our constant for an AND operation
+	BoolOperatorAnd BoolOperator = "and"
+
+	// BoolOperatorOr is our constant for an OR operation
+	BoolOperatorOr BoolOperator = "or"
```

Comment:
FYI, only renamed this because it seemed weird having just an `Op()` accessor or having an `Operator()` accessor return a `BoolOp`. Otherwise would have left it.